adapters/mobilefuse/mobilefuse.go

```diff
@@ -187,9 +194,14 @@ func (adapter *MobileFuseAdapter) getValidImps(bidRequest *openrtb2.BidRequest,
 	return validImps
 }
 
-func (adapter *MobileFuseAdapter) getBidType(imp_id string, imps []openrtb2.Imp) openrtb_ext.BidType {
-	if imps[0].Video != nil {
-		return openrtb_ext.BidTypeVideo
+func (adapter *MobileFuseAdapter) getBidType(bid openrtb2.Bid) openrtb_ext.BidType {
+	if bid.Ext != nil {
+		var bidExt BidExt
+		err := json.Unmarshal(bid.Ext, &bidExt)
+
+		if err == nil && bidExt.Mf.MediaType == "video" {
+			return openrtb_ext.BidTypeVideo
+		}
```

Comment:
Nice. Curious, do have plans to support OpenRTB 2.6's `mtype` field?

Reply:
We certainly plan to eventually but haven't seen much in the way of support from our supply or demand partners yet so it's not currently a priority.